ansible/rke2/README.md

```diff
@@ -14,14 +14,16 @@ This playbook deploys an RKE2 Kubernetes cluster.
 ## Installation
 
 1.  **Install Ansible and its dependencies:**
-
     ```bash
-    pip install --include-deps ansible
-    pip install ansible-inventory-terraform
+    ansible-galaxy collection install cloud.terraform
```

Comment:
Question here,

Do you think the same applies here to run this standalone?

```
*   **Ansible:** Version 2.9 or later is recommended.
*   **Python 3.11:**  Along with the `ansible` and `kubernetes` packages.
```

From the Rancher playbook?